receiver/podmanreceiver/logs.go

```diff
@@ -0,0 +1,124 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !windows
+// +build !windows
+
+package podmanreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/podmanreceiver"
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+
+	"go.opentelemetry.io/collector/model/pdata"
+	"go.uber.org/zap"
+)
+
+func translateEventsToLogs(logger *zap.Logger, event event) (pdata.Logs, error) {
+	ld := pdata.NewLogs()
+	rl := ld.ResourceLogs().AppendEmpty()
+	ill := rl.InstrumentationLibraryLogs().AppendEmpty()
+
+	logRecord := ill.Logs().AppendEmpty()
+	logRecord.SetName(event.Action)
+	logRecord.SetTimestamp(pdata.Timestamp(event.TimeNano))
+	logRecord.Attributes().InsertString("contianer.id", event.ID)
+
+	bodyString := fmt.Sprintf("podman %s ( %s ) %s", event.Type, event.Actor.Attributes["name"], event.Action)
+	body := pdata.NewAttributeValueString(bodyString)
+	body.CopyTo(logRecord.Body())
+
+	keys := make([]string, 0, len(event.Actor.Attributes))
+	for k := range event.Actor.Attributes {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	for _, key := range keys {
+		val := event.Actor.Attributes[key]
+		attrValue, err := convertInterfaceToAttributeValue(logger, val)
+		if err != nil {
+			return pdata.NewLogs(), err
+		}
+		if key == "image" {
+			logRecord.Attributes().Insert("container.image.name", attrValue)
```

Comment:
Could use the convention helper here and for `container.name`

Reply:
Done